test/stream.1.test.js

```diff
@@ -61,22 +62,22 @@ test('should trigger the onSend hook', t => {
   })
 
   fastify.addHook('onSend', (req, reply, payload, done) => {
-    t.ok(payload._readableState)
+    t.assert.ok(payload._readableState)
     reply.header('Content-Type', 'application/javascript')
     done()
   })
 
-  fastify.inject({
+  const response = await fastify.inject({
     url: '/'
-  }, (err, res) => {
-    t.error(err)
-    t.equal(res.headers['content-type'], 'application/javascript')
-    t.equal(res.payload, fs.readFileSync(__filename, 'utf8'))
-    fastify.close()
   })
+
+  t.assert.equal(response.statusCode, 200)
+  t.assert.equal(response.headers['content-type'], 'application/javascript')
+  t.assert.equal(response.payload, fs.readFileSync(__filename, 'utf8'))
```

Comment:
Lets use `strictEqual` here
```suggestion
  t.assert.strictEqual(response.statusCode, 200)
  t.assert.strictEqual(response.headers['content-type'], 'application/javascript')
  t.assert.strictEqual(response.payload, fs.readFileSync(__filename, 'utf8'))
```